sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnInvokers.java

```diff
@@ -179,20 +176,23 @@ public String subclass(TypeDescription.Generic superClass) {
   private static Implementation delegateOrNoop(DoFnSignature.DoFnMethod method) {
     return (method == null)
         ? FixedValue.originType()
-        : new SimpleMethodDelegation(method.targetMethod());
+        : new DoFnMethodDelegation(method.targetMethod());
   }
 
-  /** Implements an invoker method by delegating to a method of the target {@link DoFn}. */
-  private abstract static class DoFnMethodDelegation implements Implementation {
+  /**
+   * Implements an invoker method ("instrumented method") by delegating to a method of the target
```

Comment:
`Implements a method on the `DoFnInvoker` implementation (the "instrumented method") by delegating to a "target method" of the {@link DoFn}.`

(invoker method was a little ambiguous)